crew

```diff
@@ -991,7 +943,7 @@ def build_command (args)
   args["<name>"].each do |name|
     @pkgName = name
     search @pkgName
-    print_current_package
+    print_current_package true
```

Comment:
This should be `print_current_package @opt_verbose` instead.

Reply:
Done